### DISCUSSION TITLE
URLs for github.comunity seem to no longer work.

### DISCUSSION THREAD

**Discussion by bzvestey** [community/community#27372] [Discussions] [answered]
Hello, I am trying to find an old discussion at https://github.community/t/tests-that-use-the-network-fail-under-node-v18-only-in-github-action/246205 but it just sends me to https://github.com/orgs/community/discussions and I can't seem to find it at all. There are a number of links like this that I have seen today that don't seem to work any more and don't show up in the search when looking for them.

**Comment by radar-blip** [comment]
Well thats interesting. Best guess, some kind of bug.
You can file a bug report here: 
https://support.github.com/contact/bug-report

**Reply by bzvestey** [reply]
I have submitted a ticket, thank you.

**Reply by radar-blip** [reply]
Welcome.

**Reply by MrSarno** [reply]
Do you mind if I asked why you suggest they send in a ticket?

GitHub got rid of their Discourse forum. Obviously links to a forum which no longer exists won't work. That isn't a bug...

**Reply by radar-blip** [reply]
No I dont mind. It seemed like a bug to me at the time.

**Comment by melvincwng** [comment]
@bzvestey, GitHub Community Forums (github.community) has been closed and shifted over here to GitHub Community Discussions.

That probably explains why the links aren't working anymore.

Hope this clarifies.

**Reply by radar-blip** [reply]
Checked in both places, broken link.

**Comment by MrSarno** [comment, marked as answer]
The GitHub team shut down their Discourse forum; they didn't import forum threads from it when doing so.

They've posted about this at #21279.

**Reply by airtower-luna** [reply]
> they didn't import forum threads from it when doing so.

They actually imported the threads, those are here now. But apparently they didn't set up redirects for each individual thread, just for "any `github.community` URL".

**Reply by jfullstackdev** [reply]
I do agree, even the count solutions are here now, like mine I have 105 solutions, although when I started here recently it was just 4 

![image](https://user-images.githubusercontent.com/47092464/181499514-821c59ff-be89-4139-8df5-b0ecdfd57d61.png)

**Comment by bzvestey** [comment]
I guess then that the real question is if that information was backed up anywhere that we can still access it.

**Reply by jfullstackdev** [reply]
yes, as you can see in reply above, I can call it migration, I'm seeing some of those, maybe the migration is not yet complete because it's too many but will be completed

**Reply by bzvestey** [reply]
I do feel like i see more today than I did yesterday. I wish there was more communication provided around what we could expect from that, but it is what it is.

**Comment by Sytten** [comment]
What I wished is that the url would also resolve to the right discussion if they do end up migrating the data.
Currently all the links on google are broken and it's really not a great experience, they could have put the discourse in read-only mode until they did the migration...

**Reply by jfullstackdev** [reply]
it was answered here, 

https://github.com/orgs/community/discussions/27346#discussioncomment-3273455

only those with marked answers, the rest will have just the redirection

**Comment by antdking** [comment]
This decision has honestly lost me a fair amount of time, as they've just broken support for more sophisticated search engines. And it's only Day 2 (not counting weekends)!

Usually when I'm looking for problems, they're unresolved issues with bugs on the Github Platform itself. That means no "Accepted Answer", but with a wealth of information from what others have tried.

Often, problems can be solved in one context, but not the other.
Accepted Answer is a pretty poor indicator for what is valuable, because it's applicable **only** to the original question, and frequently overlooks the core problem.

To date, I've yet to click on a link to the new github community and have it take me to an actual thread.

Please fix this regression!
